### PR TITLE
[FEMDEM] Updating App after summer

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
Several minor changes to adapt the changes done in summer.
PD: One change in Solid must be merged before merging this PR

### CODE REVIEW

**Comment by AlejandroCornejo** [comment]
Waiting for https://github.com/KratosMultiphysics/Kratos/pull/5418 to be merged

File: applications/FemToDemApplication/python_scripts/MainFemDem.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def AddProcesses(self):
 		if( self.ProjectParameters.Has("output_process_list") ):
 			process_parameters.AddValue("output_process_list", self.ProjectParameters["output_process_list"])
 
-		return (process_handler.ProcessHandler(self.Model, process_parameters))
+		return (KratosMultiphysics.SolidMechanicsApplication.process_handler.ProcessHandler(self.Model, process_parameters))
```
Comment (loumalouomega) [inline applications/FemToDemApplication/python_scripts/MainFemDem.py:133]: Why you don't use the process factory from the core?

File: applications/SolidMechanicsApplication/python_scripts/process_handler.py [hunk 2]
```diff
@@ -158,11 +159,9 @@ def LoadsList(self):
     #
     def ConstructProcess(self, process):
 
-        kratos_module = __import__(process["kratos_module"].GetString())
-        python_module = __import__(process["python_module"].GetString())
+        python_module = import_module(process["kratos_module"].GetString() + "." + process["python_module"].GetString())
```
Comment (loumalouomega) [inline applications/SolidMechanicsApplication/python_scripts/process_handler.py:162]: This should use the process factory from the core... I cannot approve this anyway, I am not involved in the Solid app

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Processes should be created with the process factory of the core

**Comment by loumalouomega** [comment]
Merge this with the master and I approve

**Comment by AlejandroCornejo** [comment]
the other PR has been merged @loumalouomega

**Review by loumalouomega — APPROVED** [review]

**Pull request merged**